core/templates/pages/email-dashboard-pages/email-dashboard-page.controller.spec.ts

```diff
@@ -15,6 +15,7 @@
 /**
  * @fileoverview Unit tests for the email dashboard page.
  */
+import { UpgradedServices } from 'services/UpgradedServices';
```

Comment:
I don't think we need a TODO here. Since all spec files are being moved to Angular and this will automatically go away then.

Yeah, a newline on top should be added. I'll add that.